compose.yml

```diff
@@ -35,6 +29,11 @@ services:
       EPICS_PVAS_INTF_ADDR_LIST: "0.0.0.0"
     volumes:
       - channelfinder-es-data:/usr/share/elasticsearch/data
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9200/_cluster/health"]
```

Comment:
Should maybe check for what level of health, elastic has red, yellow and green

Reply:
I did look into that very briefly for the integration tests (don't think we care here), but decided to not scope creep further and ignore it for now